build.sbt

```diff
@@ -56,18 +63,18 @@ val relocationPrefix = s"$myPackage.repackaged"
 
 libraryDependencies ++= Seq(
   // Keep com.google.cloud dependencies in sync
-  "com.google.cloud" % "google-cloud-bigquery" % "1.82.0",
-  "com.google.cloud" % "google-cloud-bigquerystorage" % "0.98.0-beta",
+  "com.google.cloud" % "google-cloud-bigquery" % "1.96.0",
+  "com.google.cloud" % "google-cloud-bigquerystorage" % "0.114.0-beta",
   // Keep in sync with com.google.cloud
-  "io.grpc" % "grpc-netty-shaded" % "1.22.1",
+  "io.grpc" % "grpc-netty-shaded" % "1.23.0",
+  "com.google.guava" % "guava" % "28.1-jre",
 
   "org.apache.avro" % "avro" % "1.8.2",
 
   // Brings SLF4J 1.7.25. Spark 2.2 has 1.7.16
-  // TODO(pmkc): Shade
   "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2",
 
-
+  "com.google.cloud.bigdataoss" % "gcs-connector" % "hadoop2-2.0.0" % "test",
```

Comment:
Will it be enough `gcsio` or `google-cloud-storage` dependencies for tests? `gcs-connector` pretty heavy-weight because it brings Hadoop with it.

Reply:
As I'm using the HDFS API to access GCS (needed for the Spark parquet/orc writing), I've changed the scope to "runtime", and added a verification that the GCS connector is installed.

Reply:
I would not list it as runtime OR validate it. Just leave it test and let it fail.

"no filesystem for scheme gs" is a clean failure and it should fail early enough that you don't mutate any state.

Reply:
Makes sense. Fixed in https://github.com/GoogleCloudPlatform/spark-bigquery-connector/pull/66/commits/5b5d2c14c49e76e857bbe1019ae06d51cd811720